.github/config/out_of_tree_extensions.cmake

```diff
@@ -67,6 +67,15 @@ if (NOT MINGW)
             )
 endif()
 
+################# INET
+duckdb_extension_load(inet
+    DONT_LINK LOAD_TESTS
```

Comment:
`DONT_LINK` can be removed, we aim to test static linking for all extensions, only setting `DONT_LINK` if static linking is broken for whatever reason

The reason we test linking all extensions into DuckDB because it makes it easier for people to create custom builds linking in the core extensions statically.